app/src/components/common/bar_diagram_probabilities/index.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { HTMLAttributes } from 'react'
+import styled from 'styled-components'
+
+interface StyleProps {
+  isGreaterThanFifty?: boolean
+  barPercentage?: number
```

Comment:
`percentage`, the context is `bar` already.

Reply:
Update `barPercentage` to `percentage`